packages/api/src/routes/medical/patient.ts

```diff
@@ -118,25 +116,15 @@ router.put(
     }
 
     const facilityId = getFacilityIdOrFail(patient, facilityIdParam);
-
     const patientUpdate: PatientUpdateCmd = {
       ...schemaUpdateToPatient(payload, cxId),
       ...getETag(req),
       id,
+      facilityId,
     };
 
     const updatedPatient = await updatePatient(patientUpdate);
 
-    // temp solution until we migrate to FHIR
-    const fhirPatient = toFHIR(updatedPatient);
-    await upsertPatientToFHIRServer(updatedPatient.cxId, fhirPatient);
-
-    // TODO: #393 declarative, event-based integration
-    // Intentionally asynchronous - it takes too long to perform
-    cwCommands.patient
-      .update(updatedPatient, facilityId)
-      .catch(processAsyncError(`cw.patient.update`));
```

Comment:
Moved to inside `updatePatient` so the command does all its needed when we update a patient